backend/app/views/spree/admin/shared/_account_nav.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if try_spree_current_user %>
+  <ul class="nav navbar-nav">
+    <li class="dropdown user user-menu">
+      <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+        <i class="glyphicon glyphicon-user"></i>
+        <span><%= try_spree_current_user.try(:email) %> <i class="caret"></i></span>
```

Comment:
why use `try` here when you've already validated that the object exists on line 1 (above)?